### ISSUE TITLE
assets missing amazon s3

### ISSUE THREAD

**Issue by rustanacexd** [agconti/cookiecutter-django-rest#533]
http://take.ms/wSESJ

any clue? bucket is created in s3 but it is empty.

**Comment by agconti** [comment]
Hey @rustanacexd the fix in #534 should clear up this issue for you.  

Running `collectstatic` depends on the production settings to be specified for the files to be uploaded to s3.  Commands with manage.py default to local development, so this needs to be explicitly set by environmental variables for `collecstatic` to work as expected. 

You can use this command to update your existing app:
```bash
heroku config:set DJANGO_CONFIGURATION="Production" \
  DJANGO_SETTINGS_MODULE="{{cookiecutter.app_name}}.config" \
  --app {{ your app }}
```
Let me know if it solves it! ✨

**Comment by rustanacexd** [comment]
so fix #534  should clear up this issue for new installs from this cookiecutter?

**Comment by agconti** [comment]
@rustanacexd yup!
